### PR TITLE
Fix: database migrations

### PR DESCRIPTION
**Pull request by MuhammedMagdyy**
##### Why do we need this change?
This PR is for fixing database migrations
- renamed column photoURL to image.

### CODE REVIEW

File: src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql [hunk 1]
```diff
@@ -6,5 +6,5 @@
 
 */
 -- AlterTable
-ALTER TABLE "User" DROP COLUMN "image",
-ADD COLUMN     "image" TEXT NOT NULL;
+ALTER TABLE "User" RENAME COLUMN "photoURL" TO "image";
+ALTER TABLE "User" ALTER COLUMN "image" SET NOT NULL;
```
Comment (msherifs) [inline src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:10]: you have to supply a default value when adding a not null constraint

Comment (MuhammedMagdyy) [reply src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:10]: Why add a default value while it's required in the schema?

Comment (msherifs) [reply src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:10]: when you alter a table and add a not null, this constraint will be applied on the table instantly, so if there are nulls in the existing data the migration will fail

Comment (msherifs) [reply src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:10]: Postgres as far as i remember will set the null values with the default one, in other engines you have to update the table first before adding the constraint

Comment (MuhammedMagdyy) [reply src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:10]: Hmmm, So what should I do? it's the first time to do this with Prisma :"D

Comment (MuhammedMagdyy) [reply src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:10]: The table is up to date, and it works fine locally

Comment (MuhammedMagdyy) [reply src/database/prisma/migrations/20230813133033_photo_url_to_image/migration.sql:11]: Done

**Review by msherifs — APPROVED** [review]

**Review by mohamadelhinamy — APPROVED** [review]

**Pull request merged**